forge-game/src/main/java/forge/game/ability/effects/RingTemptsYouEffect.java

```diff
@@ -16,7 +16,7 @@ public class RingTemptsYouEffect extends EffectEffect {
 
     @Override
     protected String getStackDescription(SpellAbility sa) {
-        return "The Ring tempts " + sa.getActivatingPlayer() + ".";
+        return Localizer.getInstance().getMessageorUseDefault("lblTheRingTempts", "The Ring tempts") + " " + sa.getActivatingPlayer() + ".";
```

Comment:
Probably use Message parameter: like `getMessage("the ring tempts %s", playerName)`